pkg/jaeger/query/metrics.go

```diff
@@ -0,0 +1,54 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package query
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/promscale/pkg/telemetry"
+	"github.com/timescale/promscale/pkg/util"
+)
+
+var (
+	traceRequestsExec = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: util.PromNamespace,
+		Subsystem: "trace",
+		Name:      "query_requests_executed_total",
```

Comment:
It would be bad to name the metric as `telemetry_traces_queries_executed`. This looks mostly like a deisgn discussion that needs to be done at the doc level. The column names were kept as per design doc. But one thing we can do is

key name of metadata table = column name of instance information table = metric name

Updating according to this.